app/controllers/v0/sessions_controller.rb

```diff
@@ -49,14 +66,18 @@ def saml_logout_callback
     end
 
     def saml_callback
+      raise Common::Exceptions::RoutingError, params[:path] if session_activity.blank?
```

Comment:
The FE might need handling for this type of RoutingError. Also instrumentation and being mindful of this occurring in Sentry. I don't think this is a problem, but cc: @annaswims